tests/vmi_lifecycle_test.go

```diff
@@ -352,7 +353,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				vmi = addBootOrderToDisk(vmi, "disk0", &alpineBootOrder)
 				vmi = addBootOrderToDisk(vmi, "disk2", &cirrosBootOrder)
 				By("starting VMI")
-				vmi = tests.RunVMIAndExpectLaunch(vmi, 60)
+				vmi = tests.RunVMIAndExpectLaunch(vmi, 90)
```

Comment:
nit: maybe use `startupTimeout` here?

```suggestion
				vmi = tests.RunVMIAndExpectLaunch(vmi, 2*startupTimeout)
```